modules/oc-mirror-support.adoc

```diff
@@ -9,6 +9,11 @@
 
 The oc-mirror plugin supports mirroring {product-title} payload images and Operator catalogs for {product-title} versions 4.10 and later.
 
+[NOTE]
+====
+On `ppc64le` and `s390x` architectures the oc-mirror plugin is only supported for {product-title} versions 4.14 and later.
```

Comment:
```suggestion
On `ppc64le` and `s390x` architectures the oc-mirror plug-in is only supported for {product-title} versions 4.14 and later.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#p

Reply:
@tmalove I know IBM Style Guide still says plug-in. However, the RH SSG changed guidance it's plugin now https://redhat-documentation.github.io/supplementary-style-guide/#plugin-adj